sklearn/ensemble/forest.py

```diff
@@ -191,10 +191,8 @@ def fit(self, X, y, sample_weight=None):
         self : object
             Returns self.
         """
-        # Convert data
-        # ensure_2d=False because there are actually unit test checking we fail
-        # for 1d. FIXME make this consistent in the future.
-        X = check_array(X, dtype=DTYPE, ensure_2d=False, accept_sparse="csc")
```

Comment:
@amueller I removed `ensure_2d=False` here and no unit test check failed contrary to what the FIXME inline comment says. Do remember which tests used to fail?

 This change is technically not required for this PR. I can revert it if you prefer to deal with this case in another PR such as the one tackling #4252 for instance.